Startup.cs

```diff
@@ -36,24 +38,30 @@ public Startup(IHostingEnvironment hostingEnvironment)
 		public void ConfigureServices(IServiceCollection services)
 		{
 			services.AddApplicationInsightsTelemetry(_configuration);
-			services.AddMvcCore();
+			services.AddMvc();
+
+			services.Configure<RazorViewEngineOptions>(options => options.ViewLocationExpanders.Add(new HTMLLocationRemapper()));
 		}
 
 		public void Configure(IApplicationBuilder applicationBuilder, IHostingEnvironment hostingEnvironment, ILoggerFactory loggerFactory)
 		{
 			loggerFactory.AddConsole(minLevel: LogLevel.Warning);
 
-			// custom middleware to re-route all paths starting with `/app` to `/index.html`
-			applicationBuilder.Use(async (context, next) =>
-			{
-				if (context.Request.Path.StartsWithSegments(new PathString("/app")))
-					context.Request.Path = new PathString("/index.html");
-				await next();
-			});
 			applicationBuilder.UseApplicationInsightsRequestTelemetry();
 			applicationBuilder.UseApplicationInsightsExceptionTelemetry();
+			applicationBuilder.UseMvc();
 			applicationBuilder.UseDefaultFiles();
-			applicationBuilder.UseStaticFiles();
+			applicationBuilder.UseStaticFiles();
 		}
 	}
+
+	public class HTMLLocationRemapper : IViewLocationExpander
+	{
+		public IEnumerable<string> ExpandViewLocations(ViewLocationExpanderContext context, IEnumerable<string> viewLocations)
+		{
+			return viewLocations.Select(f => f.Replace("/Views/{1}/{0}.cshtml", "/client/{0}.html"));
+		}
+
+		public void PopulateValues(ViewLocationExpanderContext context) { }
+	}
```

Comment:
This file once again has inconsistent line endings.  :/

If you catch me online over the weekend, I can show you some tools that will make it much easier to catch and deal with these issues.  Unfortunately, the Visual Studio git integration doesn't provide very good tooling for noticing these sorts of things and dealing with them.